services/src/test-library/src/main/java/org/openkilda/testing/service/otsdb/OtsdbQueryService.java

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.testing.service.otsdb;
+
+import org.openkilda.testing.service.otsdb.model.StatsResult;
+
+import java.util.Date;
+import java.util.Map;
+
+public interface OtsdbQueryService {
+    StatsResult query(Date start, Date end, String metric, Map<String, String> tags);
```

Comment:
It'd be nice to have some explanation about metric and tags fields (or at least a reference to documentation)

Reply:
fixed in https://github.com/telstra/open-kilda/pull/1422/commits/5a55eb977f344b80d4b2becbb83f6b2b26ae095d